league_rpc_linux/processes/lcu_thread.py

```diff
@@ -1,5 +1,6 @@
 import time
 from threading import Timer
+from dataclasses import dataclass, field
```

Comment:
Why is field imported here?

can't see it being used.

Reply:
Oh right , had a test with it and forgot to remove it. 
Done 👍